src/Backend/Modules/Blog/Domain/Comment.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="blog_comments")
+ * @ORM\Entity(repositoryClass="Backend\Modules\Blog\Domain\CommentRepository")
+ * @ORM\HasLifecycleCallbacks
+ */
+class Comment
+{
+    /**
+     * @var int
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var int
+     * @ORM\Column(type="integer", name="post_id")
+     */
+    private $postId;
+
+    /**
+     * @var Locale
+     * @ORM\Column(type="locale", name="language")
+     */
+    private $locale;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string")
+     */
+    private $author;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string")
+     */
+    private $email;
+
+    /**
+     * @var string
```

Comment:
Nullable properties can contain null, so it should be `@var null|string`

Reply:
fixed in 183dde936c84bcfa2ac666bd0fd4db7d5a2bb1dc